cmd/hhfctl/main.go

```diff
@@ -267,6 +267,16 @@ func main() {
 							}), "failed to peer vpcs")
 						},
 					},
+					{
+						Name:  "wipe",
```

Comment:
done. since this is primarily intended for testing, maybe we could also flag this command as hidden, and users would not need to know or worry about it; let me know if you want me to make this change

Reply:
I think it's okay, IMO --yes flag gives enough "safety"